scripts/scil_evaluate_bundles_pairwise_agreement_measures.py

```diff
@@ -112,9 +113,8 @@ def load_data_tmp_saving(filename, reference, init_only=False,
         # If initilization, loading the data is useless
         if init_only:
             return None
-        density = nib.load(tmp_density_filename).get_fdata().astype(np.uint16)
-        endpoints_density = nib.load(
-            tmp_endpoints_filename).get_fdata().astype(np.uint16)
+        density = get_data_as_label(nib.load(tmp_density_filename))
```

Comment:
A density map is not labels, this is again confusing when reading the code later. From other scripts, I think you could simply switch it to float32.

Reply:
It was np.uint16 you really want to switch to float32